src/ActionList/Item.tsx

```diff
@@ -113,6 +146,12 @@ export function Item({
         <div>{text}</div>
         {description && <DescriptionContainer>{description}</DescriptionContainer>}
       </StyledTextContainer>
+      {AuxilaryIcon && (
```

Comment:
Nice work here! I love how this is consistent with the other conditional rendering (based on props) 🎊